File: hw/viteri/u1/HW10ChickenFarmSimulatorInput/HW10FarmSimulatorInputJava/GuardarPollito.java
package Guardar_Pollito;
import java.util.Scanner;
import java.io.BufferedReader;
import java.io.File;
import java.io.FileReader;
import java.io.FileWriter;
import java.io.IOException;
import java.nio.Buffer;
public class GuardarPollito {
    File pollito;
    Scanner in=new Scanner(System.in);
    String text="";
    int id=0,numerito=0;
    String name="",color="";
    boolean molting=false;
    float age=0;
    int num1=0;
    private void createFile(){
        pollito = new File("pollito.txt");
        try {
            if (pollito.createNewFile()) {
                System.out.println("File created successfully");
            }
            else{
                System.out.println("!!Error!!");
            }
        } catch (IOException exepcion) {
            exepcion.printStackTrace(System.out);
        }
    }
    private void deleteFile(){
        if (pollito.delete()) {
            System.out.println("File deleted successfully");
        }
        else{
            System.out.println("!!Error!!");
        }
    }
    private void writeToFile(){
        try {
            FileWriter write =new FileWriter(pollito,true);
                System.out.print("Id:\t");
                id=in.nextInt();
                in.nextLine();
                System.out.print("Name:\t");
                name=in.nextLine();
                System.out.print("Color:\t");
                color=in.nextLine();
                System.out.print("Age:\t");
                age=in.nextFloat();
                System.out.println("Is molting?\n1.-Yes\t\tOther Number.-No");
                numerito=in.nextInt();
                switch (numerito) {
                    case 1:
                        molting=true;
                    break;
                    case 2:
                        molting=false;
                    break;
                    default:
                        molting=false;
                    break;}
                write.write(id+"\t"+name+"\t"+color+"\t"+age+"\t"+molting+"\n");
                write.close();
            System.out.println("Text added successfully");
        } catch (IOException excepcion) {
            excepcion.printStackTrace(System.out);
        }
    }
    private void readFile(){
        String readed="";
        try {
            FileReader reader=new FileReader(pollito);
            BufferedReader reading =new BufferedReader(reader);
            while (readed!=null) {
                readed= reading.readLine();
                System.out.println(readed);
            }
        } catch (IOException excepcion) {
            excepcion.printStackTrace(System.out);
        }
    }
    public static void main(String[] args) {
        Scanner in2=new Scanner(System.in);
        int num2=0;
        GuardarPollito archivo=new GuardarPollito();
        archivo.createFile();
        do{
            archivo.writeToFile();
            System.out.println("Do you want to continue writing?\n1.- No\t\tother number.- yes");
            num2=in2.nextInt();
        }while(num2!=1);  
        archivo.readFile();    
    }
}

File: hw/viteri/u1/HW10ChickenFarmSimulatorInput/HW10FarmSimulatorInputPython/guardarpollito.py
import os

class GuardarPollito:
    def __init__(self):
        self.pollito = None
        self.text = ""
        self.id = 0
        self.numerito = 0
        self.name = ""
        self.color = ""
        self.molting = False
        self.age = 0.0
        self.num1 = 0

    def create_file(self):
        self.pollito = "pollito.txt"
        try:
            with open(self.pollito, 'w'):
                print("File created successfully")
        except IOError as e:
            print("!!Error!!:", e)

    def delete_file(self):
        try:
            os.remove(self.pollito)
            print("File deleted successfully")
        except OSError as e:
            print("!!Error!!:", e)

    def write_to_file(self):
        try:
            with open(self.pollito, 'a') as write:
                self.id = int(input("Id:\t"))
                self.name = input("Name:\t")
                self.color = input("Color:\t")
                self.age = float(input("Age:\t"))
                print("Is molting?\n1.-Yes\t\tOther Number.-No")
                self.numerito = int(input())
                self.molting = self.numerito == 1
                write.write(f"{self.id}\t{self.name}\t{self.color}\t{self.age}\t{self.molting}\n")
            print("Text added successfully")
        except IOError as e:
            print("!!Error!!:", e)

    def read_file(self):
        try:
            with open(self.pollito, 'r') as reader:
                for line in reader:
                    print(line.strip())
        except IOError as e:
            print("!!Error!!:", e)

if __name__ == "__main__":
    archivo = GuardarPollito()
    archivo.create_file()
    while True:
        archivo.write_to_file()
        num2 = int(input("Do you want to continue writing?\n1.- No\t\tother number.- yes\n"))
        if num2 == 1:
            break
    archivo.read_file()